src/components/productComponents/infoComponent.tsx

```
@@ -84,17 +92,24 @@ export const InfoComponent = () => {
+                                            .split("\n")
+                                            .map((item) =>
+                                                item.replace(/^-/, "").trim(),
+                                            )
+                                            .filter(Boolean)
+                                            .map((item, index) => (
+                                                <li
+                                                    key={index}
+                                                    className="leading-relaxed"
+                                                    dangerouslySetInnerHTML={{
+                                                        __html: item.replace(
+                                                            /<strong>(.*?)<\/strong>/g,
```

Biome (1.9.4)

[error] Avoid passing content using the dangerouslySetInnerHTML prop.

Setting content using code can expose users to cross-site scripting (XSS) attacks

(lint/security/noDangerouslySetInnerHtml)